client/component/comment.js

```diff
@@ -493,15 +494,15 @@ export function BasicComments({about=null, showInput=true, canPost=true}) {
 
 
 
-export function ComposerScreen({about, commentKey=null, intro=null}) {
+export function ComposerScreen({about, commentKey=null, intro=null, preselectedPerspectiveText=null}) {
```

Comment:
Let's just grab all the composer screen params, and pass them all down.

Reply:
done